code/modules/admin/callproc/callproc.dm

```diff
@@ -1,3 +1,96 @@
+
+GLOBAL_DATUM_INIT(AdminProcCallHandler, /mob/proccall_handler, new())
+GLOBAL_PROTECT(AdminProcCallHandler)
+
+/// Used to handle proccalls called indirectly by an admin (e.g. tgs, circuits).
+/// Has to be a mob because IsAdminAdvancedProcCall() checks usr, which is a mob variable.
+/// So usr is set to this for any proccalls that don't have any usr mob/client to refer to.
```

Comment:
dont you use /** for these kinds of autodoc comments

Reply:
I think you can do it both ways. I've seen mothblocks do it like this

Reply:
i always prefer multiple `///` just as a side effect of programming a lot in rust